src/App.js

```diff
@@ -1,13 +1,73 @@
 import React from 'react';
+import TodoForm from "./components/TodoComponents/TodoForm";
+import TodoList from "./components/TodoComponents/TodoList";
+
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
+  constructor() {
+    super();
+    this.state = {
+        item: '', //stores values passed into input
+        todos:[
+          {
+            task: 'Eat Food',
+            id: 1528817077286,
+            completed: false
+          },
+          {
+            task: 'Drink Coffee',
+            id: 1528817084358,
+            completed: false
+          }
+        ], //stores values added to todo list
+    };
+  }
+
+  //HANDLERS
+
+  //input handler
+  onChange = (event) => {
+    this.setState({[event.target.name]: event.target.value});
+  }
+
+  //add handler
+  onSubmit = (event) => {
+    event.preventDefault();
+    const todos = this.state.todos.slice()
+    todos.push(
+      {task:this.state.item, completed: false, id: Date.now()});
+      this.setState({todos, item: ""});
+  }    
+
+  toggleComplete = id => {
```

Comment:
looking great here. remember to comment a bit more to get in to good code style practice

Reply:
thanks. I'll be working on making this one functional today.